Drawsana/DrawsanaView.swift

```diff
@@ -113,6 +115,13 @@ public class DrawsanaView: UIView {
   /// you want it to look.
   public let selectionIndicatorView = UIView()
 
+  /// Layer that backs `DrawsanaView.selectionIndicatorView`. You may set this
+  /// layer's properties to change its visual apparance. Its `path` and `frame`
+  /// properties are managed by `DrawsanaView`.
+  public var selectionIndicatorViewShapeLayer: CAShapeLayer {
```

Comment:
Perhaps it makes more sense to make this a `public let`, and then capture the `selectionLayer` in `commonInit`?

Although this is a computed property, clients could in effect manipulate this variable by changing `selectionIndicatorView.layer.sublayers`.

Reply:
Good point. This was a pretty low-effort change; I just needed to quickly expose something, and this is how I happened to be accessing it at the time. That would be a good improvement to make in the future.